tools/wpt.ts

```diff
@@ -351,8 +350,8 @@ async function update() {
 
   const currentExpectation = getExpectation();
 
-  for (const path in resultTests) {
-    const { passed, failed, testSucceeded } = resultTests[path];
+  for (const result of Object.values(resultTests)) {
```

Comment:
Does that mean that `deno lint` will now warn on all usage of `for ... in ...` in user code?

Reply:
No. The `guard-for-in` rule is not tagged recommended and should have no effect unless explicitly enabled.